eslint/eslint-mattermost.json

```diff
@@ -185,7 +185,7 @@
     ],
     "multiline-ternary": [
       1,
-      "never"
+      "always-multiline"
```

Comment:
Is it the same as web? If not revert

Reply:
Checked the other repos, they're set to `never`. But we are using multiline in quite a lot of places, so I think we should revisit this.